apps/taiga-lumbermill/src/dashboards/crypto/components/minter/minter-deploy/minter-deploy.component.ts

```diff
@@ -0,0 +1,101 @@
+import {CommonModule} from '@angular/common';
+import {
+    ChangeDetectionStrategy,
+    Component,
+    EventEmitter,
+    inject,
+    Output,
+} from '@angular/core';
+import {FormArray, FormControl, FormsModule, ReactiveFormsModule} from '@angular/forms';
+import {TuiAutoFocus} from '@taiga-ui/cdk';
+import {TuiAlertService, TuiButton, TuiDialog} from '@taiga-ui/core';
+import {TuiAvatar, TuiPushService} from '@taiga-ui/kit';
+import {TuiCell} from '@taiga-ui/layout';
+import {TuiInputModule, TuiInputNumberModule} from '@taiga-ui/legacy';
+
+import type {TokenMinter} from '../minter.component';
+import {MinterService} from './minter-deploy.service';
+
+@Component({
+    standalone: true,
+    selector: 'lmb-minter-deploy',
+    imports: [
+        CommonModule,
+        FormsModule,
+        ReactiveFormsModule,
+        TuiAutoFocus,
+        TuiAvatar,
+        TuiButton,
+        TuiCell,
+        TuiDialog,
+        TuiInputModule,
+        TuiInputNumberModule,
+    ],
+    templateUrl: './minter-deploy.component.html',
+    styleUrl: './minter-deploy.component.less',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MinterDeployComponent {
+    protected readonly minterService = inject(MinterService).minterData;
+    protected readonly push = inject(TuiPushService);
+    protected readonly alert = inject(TuiAlertService);
+    protected minterForm = new FormArray(
+        this.minterService.map((item) => new FormControl(item.defaultValue)),
+    );
+
+    protected openIcon = false;
+    protected warningIcon = false;
+    protected success = false;
+    protected urlIcon = '';
+
+    @Output()
+    public readonly tokenMinterChange = new EventEmitter<TokenMinter>();
+
+    protected showDialog(): void {
+        this.openIcon = true;
+    }
+
+    protected deploy(): void {
```

Comment:
this is imperative-style method should be removed
use angular reactive form 
```
protected readonly form = FormGroup({
   name: new FormControl('', [validators);
   symbol: new FormControl('');
   ....
   etc
})
```

check this https://angular.dev/guide/forms/form-validation#validating-input-in-reactive-forms
and this https://taiga-ui.dev/pipes/field-error

Reply:
then on submit, if the form is valid, we can output the required object

Reply:
then no error windows will be there

Reply:
only success push

Reply:
It can be alert for urlIcon, because urlIcon isn't in the form

Reply:
it's ok, the error should highlight the field in red and the text below should say what the problem is. No need for additional window

If we needed an additional window on error, we would still do it in a different way, by checking the validity of the angular form